automatic/dropbox/tools/Helpers.psm1

```diff
@@ -0,0 +1,25 @@
+
+function Install {
+param(
+    [string] $packageName,
+    [string] $url,
+    [string] $checksum
+)
+    $installerType = 'exe'
+    $validExitCodes = @(0)
+    $silentArgs = '/S'
+    $checksumType = 'sha256';
+
+    Install-ChocolateyPackage $packageName `
+        $installerType `
+        $silentArgs `
+        $url `
+        $url `
+        -validExitCodes $validExitCodes `
+        -Checksum64 "$checksum" `
+        -ChecksumType64 $checksumType `
+        -ChecksumType "$checksumType" `
+        -Checksum "$checksum"
+}
+
+Export-ModuleMember *
```

Comment:
You haven't read up on extensions yet have you? https://chocolatey.org/docs/how-to-create-extensions